arcsi/api/show.py

```diff
@@ -336,9 +344,9 @@ def edit_show(id):
         )
 
 
-@arcsi.route("show/<int:id>", methods=["GET"])
+@arcsi.route("/show/<int:id>", methods=["GET"])
```

Comment:
Again, why is `/archon` prefix missing here?

Reply:
As I remember this endpoint is still used by the frontend or the mobile app.
I changed just those APIs which are only used by the Arcsi itself.

Reply:
I see. It could be mentioned in a comment on those occasions.

Reply:
Sure, meanwhile I searched after the usage of this exact API in the app and frontend repo and it is safe to add the `/archon` prefix here, I added the explanation comments for the other cases.